code/mob/living/carbon.dm

```diff
@@ -176,14 +176,45 @@
 #endif
 	if (src.traitHolder && src.traitHolder.hasTrait("reversal"))
 		amount *= -1
-
+	var/is_chg = ischangeling(src)
 	src.brainloss = max(0,min(src.brainloss + amount,120))
 
-	if (src.brainloss >= 120)
+	if (src.brainloss >= 220)
 		// instant death, we can assume a brain this damaged is no longer able to support life
 		src.visible_message("<span class='alert'><b>[src.name]</b> goes limp, their facial expression utterly blank.</span>")
 		src.death()
 		return
+	
+	if (src.get_brain_damage() >= 100) // Coma
+		if (!is_chg)
+			boutput(src, "<span class='alert'>You feel [pick("oddly", "suddenly", "quite")] [pick("faint", "dizzy", "numb", "dazed")].</span>")
+			src.change_misstep_chance(10)
+			if (!src.find_ailment_by_type(/datum/ailment/malady/coma))
+				if (prob(src.get_brain_damage() * -0.5))
```

Comment:
I don't understand why you invert the brain damage in your probs, it's not normally negative, is it?

Reply:
My bad, I copied from the deathchance and forgot to remove the negative. Fixed, though!